yarn.lock

```diff
@@ -2686,9 +2686,15 @@
     csstype "^3.0.2"
 
 "@types/react@^16.9.19":
+<<<<<<< HEAD
```

Comment:
Looks like this comes from a merge conflict. We can just delete the lock file and regenerate it.